Form/Type/CollapsibleCollectionType.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: sander
+ * Date: 08/04/15
+ * Time: 16:18
+ */
+
+namespace Opifer\CrudBundle\Form\Type;
+
+
+use Braincrafted\Bundle\BootstrapBundle\Form\Type\BootstrapCollectionType;
+
+class CollapsibleCollectionType extends BootstrapCollectionType
```

Comment:
I don't think it needs to both extend the BootstrapCollectionType and set the `bootstrap_collection` as a parent.

One of them will be fine